### PR TITLE
Change FormNickname to FormText

### PR DESCRIPTION
**Pull request by MartinaVilimova**
- Change the component from FormNickname to FormText
- Replacing Input elements in the profile with FormText components
- Improvements to some features in the profile (Delete the link if it is empty after editing. If the name value is empty, an error message and a rejected confirmation will appear.)

BAD MESSAGE: if we have the FormText component in the registration and on the profile page, we have to redesign the styles.

### CODE REVIEW

File: src/components/Profile/SocialNetwork/Network/index.tsx [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
+
+	const [actualAddress, setActualAddress] = useState<string | undefined>()
 
 	useEffect(() => {
 		setActualAddress(address)
 	}, [address])
 
+	const { errors, register, handleSubmit } = useForm<FormData>()
+
+	const changeText = (actualAddress: string | undefined) =>
```
Comment (agnes97) [inline src/components/Profile/SocialNetwork/Network/index.tsx:27]: There's some problem with this. :thinking: 
ActualAdress

**Review by agnes97 — COMMENTED** [review]
Well done!

I only have one bug - I found out I can add empty social media link, we should probably forbid that. :beetle: 

After our call this morning, I took liberty to add one commit to your PR and styles for Profile should now be fixed. :) For FormText, we're adding FormItem (it's parent element) outside of it's component - so for example in profile, I'm adding FormItemFullWidth while in Form, I'm adding FormItemForm. This way positioning is right, styles stay in the form but positioning is happening in the component you need to position it for, all standards are kept.

(FormText nemá rodičovský elemnt u sebe v komponentě, ale kamkoli ho přidáváš, a FormItem jsem proto rozdělila na dva - FormItemForm pro formuláře a FormItemFullWidth pro stránky. Případně tam tu šířku můžeme dát jako Props, ale teď by to mělo stačit. :sunflower: Takže přijdu do profilu, dám tam <FormItemFullWidth><FormText> a mám správně napozicováno.)

**Comment by agnes97** [comment]
Found a bug! You cannot set id's like this, because each id has to be unique. 

> Když zadáme stejné id více HTML elementům (a input může být na stránce kolikrát chceme), není unikátní a hází chybu.

bug3

File: src/components/Profile/SocialNetwork/Network/index.tsx (hunk 1 above)
Comment (MartinaVilimova) [reply src/components/Profile/SocialNetwork/Network/index.tsx:27]: It doesn't show me any mistakes, so I don't know 🤔

Comment (agnes97) [reply src/components/Profile/SocialNetwork/Network/index.tsx:27]: That might be a question for @macajind

**Review by agnes97 — APPROVED** [review]

**Pull request merged**